sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3430,6 +3430,18 @@ object GpuOverrides extends Logging {
           GpuGetJsonObject(lhs, rhs)
       }
     ),
+    expr[JsonTuple](
+      "Returns a tuple like the function get_json_object, but it takes multiple names. " + 
+        "All the input parameters and output column types are string.",
+      ExprChecks.projectOnly(
+        TypeSig.ARRAY.nested(TypeSig.STRUCT + TypeSig.STRING),
+        TypeSig.ARRAY.nested(TypeSig.STRUCT + TypeSig.STRING),
+        Seq(ParamCheck("json", TypeSig.STRING, TypeSig.STRING)),
+        Some(RepeatingParamCheck("field", TypeSig.STRING, TypeSig.STRING))),
```

Comment:
We need to mark that for the plugin we only support literal field.

```
Seq(ParamCheck("json", TypeSig.lit(TypeEnum.STRING), TypeSig.STRING))
```

This way the plugin will fall back to the CPU if it sees a non-literal "field" and it will update the docs too.

Reply:
Updated. Thank you very much!